x/crosschain/types/revert_options.go

```diff
@@ -75,6 +76,19 @@ func (r RevertOptions) GetEVMRevertAddress() (ethcommon.Address, bool) {
 	return addr, !crypto.IsEmptyAddress(addr)
 }
 
+// GetBTCRevertAddress validates and returns the BTC revert address
+func (r RevertOptions) GetBTCRevertAddress(chainID int64) (string, bool) {
```

Comment:
do we have btc chainID within cctx? Can you remove it from the args ?
```suggestion
func (r RevertOptions) GetBTCRevertAddress() (string, bool) {
```

Reply:
The receiver `r` is a `RevertOptions` struct and doesn't have chain id in it.